README.md

```diff
@@ -190,95 +184,85 @@ package my-org:my-component
 /// An example world for the component to target.
 world example {
     export hello-world: func() -> string
-}                
+}
 ```
 
 The component will export a `hello-world` function returning a string.
 
 The implementation of the component will be in `src/lib.rs`:
 
 ```rust
+cargo_component_bindings::generate!();
+
+use bindings::Example;
+
 struct Component;
 
-impl bindings::Example for Component {
+impl Example for Component {
     /// Say hello!
     fn hello_world() -> String {
         "Hello, World!".to_string()
     }
 }
-
-bindings::export!(Component);
 ```
 
-Here `bindings` is the bindings crate that `cargo component` generated for you.
+The `generate!` macro is responsible for generating the bindings to allow the
+Rust code to export what is expected of the component.
 
-The `export!` macro informs the bindings that the `Component` type exports
-all interfaces listed in `Cargo.toml`.
+It generates a Rust module named `bindings` containing the types and traits the
+correspond to the world definition.
 
 ## Usage
 
 The `cargo component` subcommand has some analogous commands to cargo itself:
 
 * `cargo component new` — creates a new WebAssembly component Rust project.
-* `cargo component add` — adds a component interface dependency to a cargo 
+* `cargo component add` — adds a component interface dependency to a cargo
   manifest file.
-* `cargo component build` — builds a WebAssembly component from a Rust project
-  using the `wasm32-wasi` target by default.
-* `cargo component doc` — generates API documentation for a WebAssembly 
-  component from a Rust project.
-* `cargo component metadata` — prints package metadata as `cargo metadata` 
-  would, except it also includes the metadata of generated bindings.
-* `cargo component check` — checks the local package and all of its dependencies
-  (including generated bindings) for errors.
-* `cargo component clippy` — same as `cargo clippy` but also checks generated 
-  bindings.
-* `cargo component update` — same as `cargo update` but also updates the 
+* `cargo component update` — same as `cargo update` but also updates the
   dependencies in the component lock file.
 * `cargo component publish` - publishes a WebAssembly component to a [warg](https://warg.io/)
   component registry.
-* `cargo component key` - manages signing keys for publishing WebAssembly 
+* `cargo component key` - manages signing keys for publishing WebAssembly
   components.
-* `cargo component wit` - manages the target WIT package of the component.
 
-More commands will be added over time.
+Unrecognized commands are passed through to `cargo` itself, but only after the 
+bindings information for component packages has been updated.
```

Comment:
I myself would find it useful to if this mentioned "build". "check", "doc", and "clippy" as examples of commands that are passed through.